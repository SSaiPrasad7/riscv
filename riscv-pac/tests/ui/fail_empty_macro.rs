#[riscv_pac::pac_enum]
#[derive(Clone, Copy, Debug, PartialEq)]
enum Interrupt {
    I1 = 1,
    I2 = 2,
    I4 = 4,
}

fn main() {}
