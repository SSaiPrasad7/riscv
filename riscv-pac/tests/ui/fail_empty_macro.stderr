error: unexpected end of input, expected `unsafe`
 --> tests/ui/fail_empty_macro.rs:1:1
  |
1 | #[riscv_pac::pac_enum]
  | ^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `riscv_pac::pac_enum` (in Nightly builds, run with -Z macro-backtrace for more info)
